cmd/scollector/collectors/processes_linux.go

```diff
@@ -62,18 +62,29 @@ func linuxProcMonitor(w *WatchedProc, md *opentsdb.MultiDataPoint) error {
 	var totalCPU int64
 	var totalVirtualMem int64
 	var totalRSSMem int64
-	for pid, id := range w.Processes {
-		file_status, e := os.Stat("/proc/" + pid)
-		if e != nil {
-			w.Remove(pid)
-			continue
+	var systemUptime int64
+	var bootTime int64
+	var uptimeIntRE = regexp.MustCompile(`(\d+)(\.+\d+)?\s(\d+)(\.+\d+)?`)
```

Comment:
I don't think this should require a Regex. Just strings.Frields (or something like that). Then parse the float and convert to int, and return the int. This can be a function then since it is also used in procstats_linux.og